h2o-py/h2o/automl/autoh2o.py

```diff
@@ -105,7 +113,19 @@ def __init__(self,
         assert nfolds >= 0, "nfolds set to " + str(nfolds) + "; nfolds cannot be negative. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable."
         assert nfolds is not 1, "nfolds set to " + str(nfolds) + "; nfolds = 1 is an invalid value. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable."           
         self.build_control["nfolds"] = nfolds 
-        self.nfolds = nfolds   
+        self.nfolds = nfolds
+
+        # Pass through to all algorithms
+        if balance_classes is True:
+            self.build_control["balance_classes"] = balance_classes
+            self.balance_classes = balance_classes
+        if class_sampling_factor is not None:
+            self.build_control["class_sampling_factor"] = class_sampling_factor
+            self.class_sampling_factor = class_sampling_factor
+        if max_after_balance_size != 5.0:
```

Comment:
Does `max_after_balance_size` get filled in to be `5.0` if a user doesn't change the parameter?

Reply:
It gets added to `build_control` only if user changes it from default.

Reply:
Sounds good.